.travis.yml

```diff
@@ -1,27 +1,46 @@
-sudo: false
+cache:
+  directories:
+    - $HOME/.composer/cache
+    - $HOME/.npm
+    - $HOME/.nvm/.cache
+    - $HOME/phpunit-bin
 
-language:
-    - php
-    - node_js
+notifications:
+  email:
+    on_success: never
+    on_failure: change
+
+language: php
 
 php:
-    - 5.4
-    - 7.0
+  - 5.6
+  - 7.4
+
+services:
+  - mysql
+
+addons:
+  apt:
+    packages:
+      - libxml2-utils
 
 env:
-    - WP_VERSION=latest WP_MULTISITE=0
-    - WP_VERSION=latest WP_MULTISITE=1
-    - WP_VERSION=trunk WP_MULTISITE=0
-    - WP_VERSION=trunk WP_MULTISITE=1
+  - WP_VERSION=latest WP_MULTISITE=0
+  - WP_VERSION=latest WP_MULTISITE=1
+  - WP_VERSION=trunk WP_MULTISITE=0
+  - WP_VERSION=trunk WP_MULTISITE=1
+
+before_install:
+  - nvm install
```

Comment:
Installing the required `npm` version with `nvm` here added roughly a minute to the build 😢